assets/config/0.5.4-coins.json

```diff
@@ -9345,6 +9345,40 @@
     "active": false,
     "currently_enabled": false
   },
+  "WHIVE": {
+    "coin": "WHIVE",
+    "name": "Whive",
+    "coingecko_id": "whive",
+    "coinpaprika_id": "whive-the-whive-protocol",
+    "nomics_id": "whive",
+    "electrum": [
+      {
+       "url": "electrumx1.cointest.com:50002",
+       "protocol": "SSL",
+        "disable_cert_verification": true
+      },
+      {
+        "url": "electrumx2.cointest.com:50002",
+        "protocol": "SSL",
+        "disable_cert_verification": true
+      },
+      {
+        "url": "electrumx1.cointest.com:50001",
+        "protocol": "TCP"
+      },
+      {
+        "url": "electrumx2.cointest.com:50001",
+        "protocol": "TCP"
+      }
+    ],
+    "explorer_url": [
+      "https://whiveexplorer.cointest.com/"
+    ],
+    "type": "UTXO",
+    "active": true,
+    "is_segwit_on": true,
```

Comment:
pls set to false

Reply:
i set both "active' and "is_segwit_on" to false ?